docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -154,6 +155,44 @@ \section{Failure Mode and Effect Analysis}
       Error messages and logging & a. Allow administrator to adjust
       permissions & SR-AR1 \newline SR-AR4 & H3-2 \\
       \hline
+      Accept Contractor Document Upload & Document is valid but is uploaded in
+      the wrong location & Poor discoverability and organization &
+      a. Human-computer interface failure &User detects &
```

Comment:
how does the user detect it

Reply:
added that they see the document is in the wrong category